test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala

```diff
@@ -64,7 +69,7 @@ class CustomsDeclareExportsConnectorIntegrationSpec
 
       val response = await(connector.createDeclaration(newDeclaration))
 
-      response shouldBe existingDeclaration
+      response mustBe existingDeclaration
```

Comment:
why `mustBe`?

Reply:
`UnitSpec` form `hmrctest` extends `Matchers`, our `UnitSpec` extends `MustMatchers` so we don't have access to `shouldBe`.

To have it we need to extends `Matchers` but IMO it doesn't make any sense to add additional `Matchers`. `mustBe` is consistent with actual approach in other tests.